deploy/hubble/manifests/controller/helm/retina/templates/agent/daemonset.yaml

```diff
@@ -189,11 +197,19 @@ spec:
             - -command
             - .\setkubeconfigpath.ps1; ./controller.exe --config ./retina/config.yaml --kubeconfig ./kubeconfig
           readinessProbe:
-            httpGet:
-              path: /metrics
-              port: {{ .Values.agent.container.retina.ports.containerPort }}
+            exec:
+              command:
+              - hubble
+              - status
             initialDelaySeconds: 15
             periodSeconds: 10
+          livenessProbe:
+            exec:
+              command:
+              - hubble
+              - status
+            initialDelaySeconds: 30
+            periodSeconds: 30
```

Comment:
How long does the hubble status cmd take to respond at scale? If the command is generally quick and fits within the 30s window, perfect, but if it's frequently greater than 30s for a healthy response then restarting the pod may do more harm than good

Reply:
Going to run some scale tests. TBC.

Reply:
Ref: https://github.com/cilium/cilium/blob/c7347199af4fe426c87fc0f5572f493559259ea6/pkg/hubble/observer/local_observer.go#L210

The status should be pretty static w.r.t. scale. Should have the ring buffer size (which is fixed) as upper limit.

Reply:
Also some adjustments needed to run scale tests for hubble atm